cardigan/config/decorators.tsx

```diff
@@ -8,23 +8,25 @@ export const ContextDecorator: FunctionComponent<PropsWithChildren> = ({
   children,
 }) => {
   return (
-    <ThemeProvider theme={theme}>
-      <GlobalStyle isFontsLoaded={true} />
-      <AppContextProvider>
-        <div className="enhanced">{children}</div>
-      </AppContextProvider>
-    </ThemeProvider>
+    <StyleSheetManager enableVendorPrefixes>
```

Comment:
As far as I could see, this was only needed for one thing (`box-decoration-break`), so I wonder if we couldn't accept the disabling the vendor prefixes and hand write them if needed. Opinions about going that way?

Reply:
I'd be happy with that.

I'd also be interested to see how much 'enableVendorPrefixes' adds to the CSS